tests/std/tests/P0429R9_flat_map/test.cpp

```diff
@@ -251,7 +252,20 @@ void test_construction() {
         assert(check_value_content(fmap, {44, 2324, 635462, 433, 5, 7}));
         static_assert(!is_constructible_v<flat_multimap<int, int>, sorted_unique_t, vector<int>, vector<int>>);
     }
-    // FIXME, test flat_multimap construction from sorted_equivalent_t
+    {
+        static_assert(!is_constructible_v<flat_map<int, int>, sorted_equivalent_t, vector<int>, vector<int>>);
+        static_assert(is_constructible_v<flat_multimap<int, int>, sorted_equivalent_t, vector<int>, vector<int>>);
+        vector<int, MyAllocator<int>> keys = {0, 1, 2, 2, 3, 4};
+        vector<int, MyAllocator<int>> vals = {44, 2324, 635462, 7, 433, 5};
+        flat_multimap fmmap(sorted_equivalent, keys, vals);
+        assert(check_key_content(fmmap, {0, 1, 2, 2, 3, 4}));
+        assert(check_value_content(fmmap, {44, 2324, 635462, 7, 433, 5},
+            {
+                {0, 1, subrange_type::equal},
+                {2, 3, subrange_type::permutation},
+                {4, 5, subrange_type::equal},
+            }));

```

Comment:
The `sorted_equivalent_t` constructor is guaranteed to move the keys and values in constant time, so this should be a full equality check. Probably worth a comment.